modules/metering-install-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-prerequisites_{context}"]
+= Prerequisites
+
+Operator Metering requires the following components:
```

Comment:
I don't think it makes sense to say "Operator Metering". What do you really mean here? The Metering Operator?

Reply:
"operator metering" is the project name "metering" is the most common way we refer to it. We also say "The Metering stack" sometimes. Referring to a specific operator is incorrect since it's not the operator that requires this stuff, but the components that the operator installs that requires it.

Reply:
I changed to "Metering requires..." to be consistent with how we refer to metering in the rest of the docs.